enigma-principal/app/src/boot_network/keys_provider_http.rs

```diff
@@ -46,10 +48,28 @@ impl TryInto<[u8; 65]> for StringWrapper {
     }
 }
 
+impl TryInto<U256> for StringWrapper {
+    type Error = Error;
+
+    fn try_into(self) -> Result<U256, Self::Error> {
+        let bytes = self.0.from_hex()?;
+        if bytes.len() != 32 {
+            return Err(RequestValueErr {
+                request: METHOD_GET_STATE_KEYS.to_string(),
+                message: "Cannot create a 32 bytes array from mismatching size slice.".to_string(),
+            }.into());
+        }
+        let mut slice: [u8; 32] = [0; 32];
+        slice.copy_from_slice(&bytes);
+        Ok(U256::from(slice.as_ref()))
```

Comment:
there's no need to copy it into an array if you then AsRef it into a slice anyway